test/case/ietf_interfaces/iface_enable_disable/test.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""
+Interface status
+
+Verify that interface status works
```

Comment:
Maybe: Verify interface status properly propagate changes when an interface is disabled and then re-enabled. Both admin-status and oper-status is verified.

Reply:
Thnx! I'll take it :)

Reply:
I think it should be propagates, plural.